scrnaseq.Rmd

```diff
@@ -145,11 +164,36 @@ param$pc_n = 10
 # Resolution of clusters; low values will lead to fewer clusters of cells 
 param$cluster_resolution=0.5
 
-# Thresholds to define differentially expressed genes 
-param$padj = 0.05
-param$log2FC = log2(2)
+# Thresholds to define marker genes
+param$marker_padj = 0.05
+param$marker_log2FC = log2(2)
+param$marker_pct = 0.25
 
-# P-value threshold for functional enrichment
+# DEG contrasts
+#
+# The parameter takes a table in form of a data.frame or an Excel file.
+# These columns are required:
+# condition_column: categorial column in the cell metadata; specify "orig.ident" for sample and "seurat_clusters" for cluster
+# condition_group1: condition levels in group 1; can also be multiple levels concatenated by the plus character; empty string = all levels not in group2
+# condition_group2: condition levels in group 2; separate multiple levels concatenated by the plus character; empty string = all levels not in group1
+#
+# These columns are optional:
+# subset_column: categorial column in the cell metadata to subset before testing; specify "orig.ident" for sample and "seurat_clusters" for cluster (default: none)
+# subset_group: subset levels; for the individual analysis of multiple levels separate by semicolons; for the joint analysis of multiple levels concatenate by the plus character (default: none)
+# assay: Seurat assay to test on; can also be a Seurat dimensionality reduction (default: RNA)
+# slot: when Seurat assay object, which slot to use (default: data)
+# padj: maximum adjusted p-value (default: 0.05)
```

Comment:
For both it is `0.05`. It gives the user more flexibility since he might use to another p-value for a different test setting (different method, more/less cells). Also, the p-value for the markers may not be interpretable since the data is not independent from the test (as discussed with Fabian). However, the p-value for other tests may be.

Reply:
Mmh still not sure we should allow different p-values...